VideoLocker/src/main/java/org/edx/mobile/base/BaseFragmentActivity.java

```diff
@@ -340,6 +414,30 @@ public void stopAnimation(View view) {
     }
 
     /**
+     * Display the banner at the top of the Activity.
+     *
+     * @param bannerType The banner type
+     */
+    @Override
+    public void showBanner(@NonNull BannerType bannerType) {
+        bannerTextView.setText(bannerType.getShortMessageRes(this));
+        bannerExpanded.setText(bannerType.getLongMessageRes(this));
+        bannerExpanded.setOnClickListener(bannerType.getClickListener());
+        banner.setVisibility(View.VISIBLE);
+        displayedBannerType = bannerType;
+    }
+
+    /**
+     * Hide the banner at the top of the Activity.
+     */
+    private void hideOfflineBanner() {
```

Comment:
Shouldn't this be a generic banner hiding function that takes the `BannerType` as an argument ?